VATSpy.dat

```diff
@@ -12801,7 +12800,7 @@ TVSC|Canouan|12.699167|-61.3425||TTZP|0
 TVSM|Mustique|12.88519|-61.1871||TTZP|0
 TVSU|Union Is./Union. - Intl|12.598611|-61.414722|UNI|TTZP|0
 TVSV|Arnos Vale|13.144444|-61.213889||TTZP|0
-TXKF|L F Wade Intl|32.364042|-64.678703|BDA|BDA-TMA|0
+TXKF|L F Wade Intl|32.364042|-64.678703|BDA|KZNY|0

```

Comment:
Shouldn't this be KSNY-BDA?

![image](https://user-images.githubusercontent.com/35482228/103493211-346eb780-4e30-11eb-8754-fffb90fec695.png)

Reply:
changed